izpack-installer/src/main/java/com/izforge/izpack/installer/gui/InstallerFrame.java

```diff
@@ -304,6 +305,9 @@ public void buildGUI()
 
         Messages messages = locales.getMessages();
         navigator.updateButtonText(messages);
+        
+        // set the frame title
+        setTitle(getTitle(installdata));
```

Comment:
Exactly, good hint, haven't even noticed.
The constructor of InstallerFrame is implicitely invoked by the picocontainer framework (dependency injection). The title parameter must be removed from the constructor and cleant up from the container here, because there is no usage for it at construction time of the `InstallerFrame` instance any longer.

This means you should also remove the line
```java
pico.addConfig("title", "title");
```
along with the constructor parameter (`String title`).

(picocontainer does some kind of guess here where it gets the "best" value for the constructor parameter, which is the "title" string here).